.Lib9c.Tests/Action/RawState.cs

```diff
@@ -0,0 +1,95 @@
+namespace Lib9c.Tests.Action
+{
+    using System;
+    using System.Collections.Immutable;
+    using System.Linq;
+    using Bencodex.Types;
+    using Libplanet;
+    using Libplanet.Action;
+    using Libplanet.Assets;
+
+    /// <summary>
+    /// An implementation of <see cref="IAccountStateDelta"/> for test. It handles states as raw like Libplanet does.
+    /// </summary>
+    public class RawState : IAccountStateDelta
+    {
+        private readonly IImmutableDictionary<string, IValue> _rawStates;
+
+        public RawState(IImmutableDictionary<string, IValue> rawStates = null)
+        {
+            _rawStates = rawStates ?? ImmutableDictionary<string, IValue>.Empty;
+        }
+
+        public IImmutableSet<Address> UpdatedAddresses =>
+            StateUpdatedAddresses.Union(UpdatedFungibleAssets.Keys).ToImmutableHashSet();
```

Comment:
Could you give me some example code for your comment? I couldn't understand yours. sorry. 😢

Reply:
Well I found your code was correct (I missed `Union()` 😅). But since `UpdatedFungibleAssets` will throw a `NotSupportedException`, this will also throw `NotSupportedException` too right?

Reply:
You are right 😀